package.json

```diff
@@ -9,11 +9,15 @@
     "commander": "^2.20.0",
     "lerna": "^3.20.2",
     "semver": "^7.3.2",
-    "shelljs": "^0.8.5"
+    "shelljs": "^0.8.5",
+    "typedoc": "^0.23.5",
+    "typescript": "^4.7.2",
```

Comment:
I'm not sure how having typescript in the root impacts the rest of the packages, but it feels like something we should be careful with / mindful of the impact.

Reply:
Typedoc specified typescript as its peerDependency, so if we were to use typed for the whole dependency, we'd have to install typescript as well. Let me know if you know another way to do it

Reply:
Is our goal to have generated docs for all packages? Can typedoc generate docs for JS packages? Do we want to use a single docs generator for all packages? If the answer to all of those is yes, then I think it makes sense to have typedoc at root. I'm not opposed to starting with typedoc scoped to a single package, but if our goal is to use it more, then I think we might as well save some effort and just put it where we know it will eventually be.

I haven't verified, but I _think_ having typescript installed at root shouldn't impact much. I think most scripts are executed from the context of each individual package, so they wouldn't look in parent directories for typescript/tsconfig.